### PR TITLE
chore(ext/node): ignore non-working process test

### PR DESCRIPTION
**Pull request by mmastrac**
Filed https://github.com/denoland/deno/issues/22724 for actual issue

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

File: tests/unit_node/process_test.ts [hunk 1]
```diff
@@ -243,7 +243,7 @@ Deno.test({
           setInterval(() => {}, 1000);
           const listener = () => {
             console.log("foo");
-            process.off("SIGINT")
+            process.off("SIGINT", listener)
```
Comment (bartlomieju) [inline tests/unit_node/process_test.ts:246]: Err, this seems off. Can you revert it?

Comment (mmastrac) [reply tests/unit_node/process_test.ts:246]: This is actually required in node.js -- the code won't run w/o it.

Comment (mmastrac) [reply tests/unit_node/process_test.ts:246]:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "listener" argument must be of type function. Received undefined
```

Comment (bartlomieju) [reply tests/unit_node/process_test.ts:253]: Huh, so it didn't fail properly the test even though the required arg was not provided?

Comment (mmastrac) [reply tests/unit_node/process_test.ts:253]: Correct. The test wasn't really testing anything. I think process.off needs a bit more work.

**Pull request merged**